src/displayapp/screens/Symbols.h

```diff
@@ -34,6 +34,8 @@ namespace Pinetime {
         static constexpr const char* hourGlass = "\xEF\x89\x92";
         static constexpr const char* lapsFlag = "\xEF\x80\xA4";
         static constexpr const char* drum = "\xEF\x95\xA9";
+        static constexpr const char* dice = "\xEF\x94\xA2";
+        static constexpr const char* chartLine = "\xEF\x88\x81";
```

Comment:
Don't think this chartLine symbol is related.

Reply:
Thank you! Removed in https://github.com/InfiniTimeOrg/InfiniTime/pull/1326/commits/3c4027ab3cf6231f7de64822db51929ab696a8e7